tests/_fal_testing/utils.py

```diff
@@ -1,29 +1,31 @@
 import os
 from pathlib import Path
-from functools import partial
+from typing import Optional
 
-def create_artifact(context, suffix):
+
+def create_artifact(context, suffix, *, output: Optional[str] = None):
     model_name = context.current_model.name
-    model_status = context.current_model.status
 
     temp_dir = Path(os.environ["temp_dir"])
     temp_file = (temp_dir / model_name).with_suffix(suffix)
 
-    output = f"Model name: {model_name}"
-    output += f"\nStatus: {model_status}"
-    output += f"\nModel dataframe name: {model_name}"
+    if not output:
+        output = output_from_context(context)
     temp_file.write_text(output)
 
 
-def create_model_artifact(context):
-    create_artifact(context, ".txt")
+def output_from_context(context):
+    model_name = context.current_model.name
+    model_status = context.current_model.status
+
+    output = f"Model name: {model_name}"
+    output += f"\nStatus: {model_status}"
+    return output
 
 
-def create_script_artifact(context, prompt):
-    create_artifact(context, f".{prompt}.txt")
+def create_model_artifact(context, **kwargs):
+    create_artifact(context, ".txt", **kwargs)
 
 
-create_before_script_artifact = partial(create_script_artifact, prompt="before")
```

Comment:
Why are we removing these? (I also decided to remove them in the structured hooks PR, but not sure why their removal is needed here)

Reply:
not needed, more of an opinion. It came from needing to replace the content created inside the artifact.

Reply:
Wdyt about the approach in the structed hooks PR (`create_dynamic_artifact`)?

Reply:
I think that works. I can remove these changes.

Reply:
Thanks!